ethereum/performance-trackers/src/main/java/tech/pegasys/teku/ethereum/performance/trackers/BlockProductionPerformance.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Consensys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.ethereum.performance.trackers;
+
+import java.util.function.Supplier;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+public interface BlockProductionPerformance {
+  String COMPLETE_LABEL = "complete";
+  int LATE_EVENT_MS = 100;
```

Comment:
We need some confidence on this value.  Though block production is very rare for home stakers and it will not make logs too spammy even with low value, is it really dangerous if it's 130ms?

Reply:
From the numbers I saw on our goerli nodes, 130 will fire 99% of the times.
So if here we just want to capture a warning, this shouldn't fire that often and I'm thinking about a 250-300 value.